### ISSUE TITLE
`$ mne make_scalp_surfaces` requires access to screen

### ISSUE THREAD

**Issue by christianbrodbeck** [mne-tools/mne-python#3417]
For me, `$ mne make_scalp_surfaces` ends with

```
...
3. Creating medium tessellation...
3.1 Decimating the dense tessellation...
This program needs access to the screen.
Please run with a Framework build of python, and only when you are
logged in on the main display of your Mac.
```
(OS X, Anacond 2.3, on master as well as 0.12). The cause seems to be `mne.decimate_surface` which calls on TVTK. Is this a known issue? The normal solution to this error would be to run a script with `pythonw` (the framework build) instead of `python` but I can't figure out how the `$ mne` command can be invoked with `pythonw`?

**Comment by agramfort** [comment]
you mean it fails without X server?

**Comment by christianbrodbeck** [comment]
Not sure what you mean, I am just running the command in the terminal. Should I run the command in a special way or install something? I have XQuartz installed if that is it.

**Comment by agramfort** [comment]
weird. tvtk is a dependency of mayavi so it should be lighter and work if
mayavi works on your system. Sorry I can't tell more

**Comment by Eric89GXL** [comment]
`wx` complains about framework builds like this. Try doing `ETS_TOOLKIT=qt4 command`

**Comment by christianbrodbeck** [comment]
Thanks @Eric89GXL that was it! Would you be opposed to adding `os.environ['ETS_TOOLKIT'] = 'qt4'` to the `$ mne make_scalp_surfaces` command as we do for the `$ mne coreg` command?

**Comment by Eric89GXL** [comment]
I don't think we should really add that, if someone wants to or needs to
use wx then they can't. It would be better to catch the error (I think it's
actually SystemExit) and say something informative

**Comment by christianbrodbeck** [comment]
So would it also be preferable to handle the environment for the GUI that way?

**Comment by Eric89GXL** [comment]
Yeah I think so

**Comment by christianbrodbeck** [comment]
addressed in #3422
